### PR TITLE
Issue a warning when add-on is already signed. (bug 1070188)

### PR DESCRIPTION
**Pull request by olivier-m**
I added a test to check if an add-on has a preexisting signature.
See: https://bugzilla.mozilla.org/show_bug.cgi?id=1070188

The test only checks the presence of signature files, not their contents.

### CODE REVIEW

**Comment by magopian** [comment]
@clouserw @mstriemer r?

**Comment by mstriemer** [comment]
I'm not sure how addon signing works. What happens if someone has a one or two of those three files?

File: validator/testcases/content.py [hunk 1]
```diff
@@ -258,6 +258,22 @@ def test_packed_scripts(err, xpi_package):
             err.pop_state()
 
 
+@decorator.register_test(tier=1)
+def test_signed_xpi(err, xpi_package):
+    """Checks if XPI is signed."""
+    search = set([
+        'META-INF/manifest.mf',
+        'META-INF/zigbert.rsa', 'META-INF/zigbert.sf'])
```
Comment (magopian) [inline validator/testcases/content.py]: Strange indentation

Comment (mstriemer) [inline validator/testcases/content.py]: Can you drop `'META-INF/zigbert.sf'` to its own line, please?

```python
    search = set([
        'META-INF/manifest.mf',
        'META-INF/zigbert.rsa',
        'META-INF/zigbert.sf',
    ])
    # or
    search = set(['META-INF/manifest.mf',
                  'META-INF/zigbert.rsa',
                  'META-INF/zigbert.sf'])
```

File: validator/testcases/content.py [hunk 2]
```diff
@@ -258,6 +258,22 @@ def test_packed_scripts(err, xpi_package):
             err.pop_state()
 
 
+@decorator.register_test(tier=1)
```
Comment (mstriemer) [inline validator/testcases/content.py]: This probably makes sense as `tier=2` for "security test".

**Comment by magopian** [comment]
@mstriemer according to a discussion with @rtilder on irc:
```17:31 < rtilder> magopian: the signed XPI's META-INF directory should have a manifest.mf, zigbert.sf, and zigbert.rsa.```

So I believe that to be properly signed, an addon needs all those three files.

**Comment by magopian** [comment]
Taking over on this PR, opened #268

**Pull request closed without merging**